### ISSUE TITLE
Dependencies on Windows

### ISSUE THREAD

**Issue by lvqcl** [yt-dlp/yt-dlp#1163]
## Checklist

- [x] I'm reporting a bug unrelated to a specific site
- [x] I've searched the bugtracker for similar bug reports including closed ones

## Description

README.md states that yt-dlp has a dependency on MSVS 2010 SP1 Redistributable Package (x86). It seems that this statement is no longer correct, probably due to deprecation of Python 3.4 and moving to Python 3.7.

**Comment by pukkandan** [comment]
Are you sure? Maybe you are using PIP? The dependency is for the exe only

**Comment by lvqcl** [comment]
I downloaded yt-dlp 2021.09.25, yt-dlp.exe, yt-dlp_x86.exe and yt-dlp.zip. All three .exe files depend on MSVCRT.DLL.
Also it seems that they need (self-contained) VCRUNTIME140.DLL.

I have MSVS 2010 redist installed in my system, so I can't test whether yt-dlp_x86.exe works without it. But it seems that MSVS 2010 redist is not required.

**Comment by pukkandan** [comment]
You are correct. What is needed is VC++14 which is seems to be already included inside the executable
https://wiki.python.org/moin/WindowsCompilers#Which_Microsoft_Visual_C.2B-.2B-_compiler_to_use_with_a_specific_Python_version_.3F
